src/scene/light.js

```diff
@@ -65,6 +65,9 @@ class LightRenderData {
         // scissor rectangle for the shadow rendering to the texture (x, y, width, height)
         this.shadowScissor = new Vec4(0, 0, 1, 1);
 
+        // depth range for cascaded shadows
+        this.depthRange = 0;
```

Comment:
this should be this.depthRange = new Vec2(); instead

Reply:
Fixed.